packages/node-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@subql/node-core",
-  "version": "5.0.4-4",
+  "version": "5.0.4",
```

Comment:
This is a breaking change because of the debug logging. It will have to be a major version. 

There were changes to the utils package too